### DISCUSSION TITLE
Urgent Help | Error: vfkit exited unexpectedly with exit code 1

### DISCUSSION THREAD

**Discussion by rashid642** [containers/podman#23576] [General]
I have macbook pro M1 2021
And whenever I try to turn on machine using "podman machine start"
It shows 

Starting machine "podman-machine-default"
Error: vfkit exited unexpectedly with exit code 1

I am unable to find any solution anywhere please help

I have tried deleting machine, and reinitilzaing it, deleting connections and making new connections, but nothing is actually working

**Comment by Luap99** [comment]
Did you install the amd64 version? 
https://blog.podman.io/2024/03/the-dreaded-vfkit-exit-1/
